diff --git a/Observer/AddToolbar.php b/Observer/AddToolbar.php
index 303a939..61b4372 100644
--- a/Observer/AddToolbar.php
+++ b/Observer/AddToolbar.php
@@ -96,7 +96,7 @@ class AddToolbar implements ObserverInterface
 
         // we do not want that the toolbar has a impact on stats => compute the stat in first
         $this->helperData->startTimer('profiler_build');
-        $this->helperProfiler->computeStats();
+        //$this->helperProfiler->computeStats();
         $this->helperData->stopTimer('profiler_build');
 
         /** @var MagentoRequest $request */
diff --git a/Observer/AddZones.php b/Observer/AddZones.php
index 05b24a6..2bdf0d2 100644
--- a/Observer/AddZones.php
+++ b/Observer/AddZones.php
@@ -56,7 +56,7 @@ class AddZones implements ObserverInterface
         MysqlFactory $mysqlBlockFactory,
         ObserverFactory $observerBlockFactory,
         PreferenceFactory $preferenceBlockFactory,
-        ProfilerFactory $profilerBlockFactory,
+//        ProfilerFactory $profilerBlockFactory,
         RequestFactory $requestBlockFactory,
         ResponseFactory $responseBlockFactory
     ) {
@@ -66,7 +66,7 @@ class AddZones implements ObserverInterface
         $this->blockFactories[] = $layoutBlockFactory;
         $this->blockFactories[] = $mysqlBlockFactory;
         $this->blockFactories[] = $cacheBlockFactory;
-        $this->blockFactories[] = $profilerBlockFactory;
+//        $this->blockFactories[] = $profilerBlockFactory;
         $this->blockFactories[] = $observerBlockFactory;
         $this->blockFactories[] = $preferenceBlockFactory;
     }
diff --git a/composer.json b/composer.json
index 33203a6..27c448a 100644
--- a/composer.json
+++ b/composer.json
@@ -1,5 +1,5 @@
 {
-    "name": "smile/module-debug-toolbar",
+    "name": "patched/smile-module-debug-toolbar",
     "type": "magento2-module",
     "description": "Smile Debug Toolbar",
     "homepage": "https://github.com/Smile-SA/magento2-module-debug-toolbar",
@@ -11,7 +11,7 @@
         }
     ],
     "license": "EPL-2.0",
-    "version": "3.1.0",
+    "version": "13.1.0",
     "require": {
         "php": "~7.0.6|~7.1.0",
         "magento/framework": "~101.0.0"
diff --git a/registration.php b/registration.php
index dc86367..697a9ca 100644
--- a/registration.php
+++ b/registration.php
@@ -7,6 +7,8 @@
  */
 if (PHP_SAPI !== 'cli') {
     // we need to declare the stat profiler manually, to use it after
+
+    /*
     $options = [
         'drivers' => [
             [
@@ -18,6 +20,7 @@ if (PHP_SAPI !== 'cli') {
 
     \Magento\Framework\Profiler::applyConfig($options, BP, false);
     \Smile\DebugToolbar\Helper\Profiler::setStat($options['drivers'][0]['stat']);
+    */
 }
 
 \Magento\Framework\Component\ComponentRegistrar::register(
